minio/minio
# use logger.LogOnce to reduce printing disconnection logs

## Description
use logger.LogOnce to reduce printing disconnection logs

## Motivation and Context
fixes https://github.com/minio/minio/issues/15334

## How to test this PR?
Run a 3-4 node distributed setup and then take down nodes and observe the logs with or without this PR. 

## Types of changes

## Checklist:
